### PR TITLE
Skios 138 use keycloak in graph ql

### PR DESCRIPTION
**Pull request by LukasLJL**
# Skiosa - PR
- [x] added test(s), or explain why one is not needed/possible...
- [x] added comments to all the functions
- [ ] added documentation to latex project
- [x] code can't be split up into smaller logical units
- [x] code is formated and linted

only postman tests, no unit tests bc I don´t know how I should test the Keycloak Middlewares
Doc will be added later bc this story is a "fix"

### CODE REVIEW

**Review by eintim — APPROVED** [review]
Looks good. Please solve conflicts.

File: src/model/jwt.ts [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
 export interface UserInfo {
   id: string | undefined;
   email: string | undefined;
   username: string | undefined;
   firstName: string | undefined;
   lastName: string | undefined;
 }
+
+@ObjectType()
+export class UserInfo implements UserInfo {
```
Comment (grossamos) [inline src/model/jwt.ts:12]: Possibly add java-doc explaining this (especially that it's a JWT).

File: src/util/middelwares.ts [hunk 2]
```diff
@@ -15,6 +18,7 @@ import { UserInfo } from "../model/jwt";
     res.json("Hello " + req.UserInfo?.username);
   }
 );
+ * @deprecated
  */
 export function getUserInfo(req: express.Request, _res: express.Response, next: express.NextFunction) {
```
Comment (grossamos) [inline src/util/middelwares.ts:23]: isn't used anywhere in our code, can be deleted without breaking backwards compatibility

**Review by grossamos — APPROVED** [review]
only minor (and very optional) things commented, merge looks good to me

File: src/util/middelwares.ts (hunk 2 above)
Comment (LukasLJL) [reply src/util/middelwares.ts:23]: Currently we use this function to test the transformer, its bad ik. For legacy reasons I would keep this deprecated Code if we need it for Express later on. If not we can delete it anyway.

File: src/model/jwt.ts (hunk 1 above)
Comment (LukasLJL) [reply src/model/jwt.ts:17]: fixed

**Pull request merged**